app/models/assignment.rb

```diff
@@ -281,7 +281,11 @@ def display_for_note
   def max_mark(user_visibility = :ta)
     # TODO: sum method does not work with empty arrays. Consider updating/replacing gem:
     #       see: https://github.com/thirtysixthspan/descriptive_statistics/issues/44
-    max_marks = get_criteria(user_visibility).map(&:max_mark)
+    if user_visibility == :peer
```

Comment:
Don't do it like this. Look for where `max_mark` is called, and call it on either the assignment or `pr_assignment` as appropriate.